src/index.test.js

```diff
@@ -0,0 +1,26 @@
+const storageNewElement = require('./index.js');
+const { default: renderElement } = require('./modules/itemFunctions.js');
+
+describe('Add function', () => {
+  expect(storageNewElement('make breackfast', false, 3)).toBE({
+    description: 'make breackfast',
+    checked: false,
+    index: 3,
+  });
+});
+
+describe('Remove function', () => {
+  expect(renderElement('make breackfast', false, 3)).toBE(
+    `
+      <li class="flexItem">
+        <span class="checked">
+          <input type="checkbox" class="checkbox" data-index=3 checked=true>
+          <p data-index=3>make breackfast</p>
+        </span>
+        <span>
+        <i class="fa-solid fa-trash-can" data-index=3></i>
+        </span>
+      </li>      
+    `,
+  );
```

Comment:
- [ ] This test doesn't test the function that removes list items from the list of tasks. Kindly add tests for functions or statements that removes a single task from the list and clears all completed tasks from the list of tasks. The test you have currently is for rendering the tasks to the DOM, kindly fix it.